backend/geolibre_server_api/geolibre_server_api/main.py

```
@@ -638,6 +673,32 @@ def get_project(
     ):
         return {"project": project_json(visible(session.get(Project, project_id), account))}
 
+    @app.get("/api/projects/{project_id}/activity")
+    def get_project_activity(
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `get_project_activity`

(ANN202)